samples/snippets/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -1502,13 +1523,35 @@ static void queryWithNumeric(DatabaseClient dbClient) {
   }
   // [END spanner_query_with_numeric_parameter]
 
+  // [START spanner_query_with_json_parameter]
+  static void queryWithJson(DatabaseClient dbClient) {
```

Comment:
It seems that this sample is added twice: Once as a standalone sample (the preferred way) and once in this monolith sample file (deprecated). I think this one should be removed.